### PR TITLE
Change Sample URLs in .NET Samples

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

Update the sample links in .NET samples from developer.tizen. to docs.tizen.

### Bugs Fixed ###

 - Issue https://github.com/Samsung/tizen-docs/issues/1269

### CODE REVIEW

File: docs/application/dotnet/samples/watch/overview.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ for TD:
             This sample application includes the watch and stopwatch functionality.</p>
 			<p>In addition, there are similar native and web sample applications.<br>
 			<ul>
-				<li><a href="https://developer.tizen.org/development/sample/native/Watch/Chronograph_Watch" target="_blank">Native version</a></li>
-				<li><a href="https://developer.tizen.org/development/sample/web/Watch/Chronograph_Watch" target="_blank">Web version</a></li>
+				<li><a href="/development/sample/native/Watch/Chronograph_Watchs" target="_blank">Native version</a></li>
```
Comment (mijin80lee) [inline docs/application/dotnet/samples/watch/overview.md:71]: Could you check the URL? 
I think the original link is 'https://docs.tizen.org/development/sample/native/Watch/Chronograph_Watch'.

Comment (mijin85cho) [reply docs/application/dotnet/samples/watch/overview.md:71]: In the docs site, the default URL is 'https://docs.tizen.org`. So the URL starting with /development will be converted into `https://docs.tizen.org/development ...`.
I think you can performance of this hyperlink on stg site, for example https://docs1.stg.tizen.org/staging/1275/application/dotnet/samples/watch/overview/.

Comment (mijin80lee) [reply docs/application/dotnet/samples/watch/overview.md:71]: I know and understand what you mean and I meant the URL is not '~Watch/Chronograph_Watchs' but '~Watch/Chronograph_Watch'.

Comment (mijin85cho) [reply docs/application/dotnet/samples/watch/overview.md]: Thanks for the reply. I updated.

**Review by journey2w — APPROVED** [review]
LGTM!

**Pull request merged**